bundles/org.openhab.core.io.rest.core/src/main/java/org/openhab/core/io/rest/core/internal/link/ItemChannelLinkResource.java

```diff
@@ -314,4 +319,45 @@ private boolean isEditable(String linkId) {
         }
         return thing.getChannel(channelUID);
     }
+
+    @GET
+    @Path("/orphans")
+    @Operation(operationId = "getOrphanLinks", summary = "Get orphan links between items and broken/non-existent thing channels", responses = {
+            @ApiResponse(responseCode = "200", description = "List of broken links") })
+    public Response getOrphanLinks() {
+        return Response.ok(getOrphanInternalLinks()).build();
+    }
+
+    public List<BrokenItemChannelLinkDTO> getOrphanInternalLinks() {
```

Comment:
I think this should be part of `ItemChannelLinkRegistry`, because the `purge` operation is already located there and we can share code between those methods.